### PR TITLE
chore: Update FVM to 3.0.0

### PR DESCRIPTION
**Pull request by riscait**
## 🙌 What's Done

- Update FVM to 3.0.0

## ✍️ What's Not Done

-

### CODE REVIEW

File: .vscode/settings.json
Comment (riscait) [inline .vscode/settings.json:1]: Indentation is set to 4 spaces.
Please turn on "Hide whitespace" when reviewing.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,104 +1,103 @@
...
+        "*.apns": "json"
+    },
+    "files.watcherExclude": {
+        "**/.fvm": true
+    },
+    "git.autofetch": true,
+    "git.enableSmartCommit": true,
+    "search.exclude": {
+        "**/*.dart_tool": true,
+        "**/*.freezed.dart": true,
```
Comment (riscait) [inline .vscode/settings.json:95]: fix.

**Review by k-nkmr — APPROVED** [review]
LGTM

**Review by naipaka — APPROVED** [review]
LGTM

**Pull request merged**